### PR TITLE
Feature/is test recipient

### PR DESCRIPTION
**Pull request by achen2401**
part of https://www.pivotaltracker.com/story/show/185649710
Allow recipient to be marked as a test recipient on the UI
[see related PR](https://github.com/uwcirg/cosri-patientsearch/pull/192)
example screenshots:

Screen Shot 2023-08-02 at 9 06 59 AM

Screen Shot 2023-08-02 at 9 05 48 AM

### CODE REVIEW

**Review by mcjustin — APPROVED** [review]
Thanks for the quick work! Perhaps we should switch to this branch for testing on dev...

File: src/model/modelUtil.ts [hunk 1]
```diff
@@ -18,3 +21,18 @@ export function makeCommunicationRequests(patient: Patient, planDefinition: Plan
...
+  patientId: string
+): Promise<any> {
+  if (!client) return null;
+  return client.request({
+    url: `Patient/${patientId}/$meta-delete`,
+    method: "POST",
+    headers: {
+      "content-type": "application/xml",
+    },
+    body: `<Parameters xmlns="http://hl7.org/fhir"><parameter><name value="meta"/><valueMeta><security><system value="${SystemURL.testPatientUrl}"/><code value="${Patient.TEST_PATIENT_SECURITY_CODE}"/></security></valueMeta></parameter></Parameters>`,
```
Comment (ivan-c) [inline src/model/modelUtil.ts:52]: I've been messing around with Parameter FHIR resources from Bulk FHIR work, and I think this JSON should be equivalent to the XML [from the example](https://www.hl7.org/fhir/resource-operation-meta-delete.html#examples)

I was able to successfully test the JSON with curl as you and Paul had (thanks for the notes!), but haven't tested the suggested changes in this PR.
```
RESOURCE='{"resourceType":"Parameters","parameter":[{"name":"meta","valueMeta":{"security":[{"system":"http://terminology.hl7.org/CodeSystem/v3-ActReason","code":"HTEST"}]}}]}'

curl -X POST -H 'Content-Type: application/json' -d "$RESOURCE" 'https://fhir.isacc.dev.cirg.uw.edu/fhir/Patient/2/$meta-delete'

{
  "resourceType": "Parameters",
  "parameter": [ {
    "name": "return",
    "valueMeta": {
      "versionId": "137",
      "lastUpdated": "2023-08-02T16:52:58.841+00:00"
    }
  } ]
}
```

```suggestion
    headers: {
      "content-type": "application/json",
    },
    body: {
      "resourceType" : "Parameters",
      "parameter" : [{
          "name" : "meta",
          "valueMeta" : {
            "security": [{
              "system": SystemURL.testPatientUrl,
              "code": Patient.TEST_PATIENT_SECURITY_CODE
            }]
          }
      }]
    },
```

**Review by ivan-c — APPROVED** [review]
Looks good! Sorry for the surprise review, just wanted to document a JSON version

File: src/model/modelUtil.ts (hunk 1 above)
Comment (achen2401) [reply src/model/modelUtil.ts:52]: thank you @ivan-c! I verified that the JSON version works as well and documented in the code.

**Pull request merged**